Assignment 1/src/main/java/com/cezaram28/Assignment1/repository/data/DataAnswerRepository.java

```diff
@@ -0,0 +1,31 @@
+package com.cezaram28.Assignment1.repository.data;
+
+import com.cezaram28.Assignment1.entity.Answer;
+import com.cezaram28.Assignment1.repository.AnswerRepository;
+import org.springframework.data.repository.Repository;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface DataAnswerRepository extends Repository<Answer, Integer>, AnswerRepository {
+    Optional<List<Answer>> findByAuthorId(int userId);
```

Comment:
Keep in mind our discussion about Optionals. You don't have to change it now (as it is too late + too much effort), but keep it in mind for the future. https://stackoverflow.com/a/30223872/7612556